test/e2e/placement_selecting_resources_test.go

```diff
@@ -634,52 +633,7 @@ var _ = Describe("validating CRP when resource selector is not valid", Ordered,
 			},
 		}
 		By(fmt.Sprintf("creating placement %s", crpName))
-		Expect(hubClient.Create(ctx, crp)).To(Succeed(), "Failed to create CRP %s", crpName)
-	})
-
-	AfterAll(func() {
-		By(fmt.Sprintf("deleting placement %s", crpName))
-		cleanupCRP(crpName)
-	})
-
-	It("should update CRP status as expected", func() {
-		crpStatusUpdatedActual := func() error {
-			crp := &placementv1beta1.ClusterResourcePlacement{}
-			if err := hubClient.Get(ctx, types.NamespacedName{Name: crpName}, crp); err != nil {
-				return err
-			}
-
-			wantStatus := placementv1beta1.ClusterResourcePlacementStatus{
-				Conditions: []metav1.Condition{
-					{
-						Type:               string(placementv1beta1.ClusterResourcePlacementScheduledConditionType),
-						Status:             metav1.ConditionFalse,
-						Reason:             clusterresourceplacement.InvalidResourceSelectorsReason,
-						ObservedGeneration: crp.Generation,
-					},
-				},
-			}
-			if diff := cmp.Diff(crp.Status, wantStatus, crpStatusCmpOptions...); diff != "" {
-				return fmt.Errorf("CRP status diff (-got, +want): %s", diff)
-			}
-			return nil
-		}
-		Eventually(crpStatusUpdatedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update CRP %s status as expected", crpName)
-	})
-
-	It("can delete the CRP", func() {
-		// Delete the CRP.
-		crp := &placementv1beta1.ClusterResourcePlacement{
-			ObjectMeta: metav1.ObjectMeta{
-				Name: crpName,
-			},
-		}
-		Expect(hubClient.Delete(ctx, crp)).To(Succeed(), "Failed to delete CRP %s", crpName)
-	})
-
-	It("should remove controller finalizers from CRP", func() {
-		finalizerRemovedActual := allFinalizersExceptForCustomDeletionBlockerRemovedFromCRPActual()
-		Eventually(finalizerRemovedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to remove controller finalizers from CRP %s", crpName)
+		Eventually(hubClient.Create(ctx, crp), eventuallyDuration, eventuallyInterval).ShouldNot(Succeed(), "Validation webhook did catch invalid cluster resource in crp", crpName)
```

Comment:
you don't need the eventually. The create should fail immediately + please check the error string,

eg,

err := hubClient.Create(ctx, crp)
		var statusErr *k8sErrors.StatusError
		Expect(errors.As(err, &statusErr)).To(BeTrue(), fmt.Sprintf("Create CRP call produced error %s. Error type wanted is %s.", reflect.TypeOf(err), reflect.TypeOf(&k8sErrors.StatusError{})))
		Expect(statusErr.Status().Message).Should(ContainSubstring(fmt.Sprintf("the labelSelector and name fields are mutually exclusive in selector %+v", selector[0])))

can you move this test to the webhook_test.go?